.github/dependabot.yml

```diff
@@ -0,0 +1,139 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/packages/flutter_plugin_android_lifecycle/example/android/app"
+    schedule:
+      interval: "weekly"
```

Comment:
FWIW, I'd suggest making the interval daily, just to spread out the dependabot PRs

Reply:
sgtm. PTAL